Sources/snowman/main.swift

```diff
@@ -2,11 +2,17 @@ import SnowManLogic
 import SnowManDictionary
 
 let dictionaryManager = DictionaryManager(words: SnowManDictionary.words) 
-let guessManager = GuessManager(totalIncorrectGuessesAllowed: 10)
 let word = dictionaryManager.getRandomWord()
 
 let io = IO()
+
 let view = View(io: io)
+
+let maxNumberOfGuesses = 10
+view.promptNumberOfGuesses()
+let numberOfGuesses = view.receiveNumberOfGuesses(maxNumberOfGuesses)
```

Comment:
That's pretty weird too....the guess manager should only be concerned with managing the guesses